packages/fuel-gauge/src/fee.test.ts

```diff
@@ -0,0 +1,269 @@
+import { generateTestWallet } from '@fuel-ts/wallet/test-utils';
+import { readFileSync } from 'fs';
+import type { BN, BaseWalletUnlocked, CoinQuantityLike } from 'fuels';
+import {
+  BaseAssetId,
+  ContractFactory,
+  FUEL_NETWORK_URL,
+  Predicate,
+  Provider,
+  ScriptTransactionRequest,
+  Wallet,
+  bn,
+} from 'fuels';
+import { join } from 'path';
+
+import callTestAbi from '../fixtures/forc-projects/call-test-contract/out/debug/call-test-abi.json';
```

Comment:
Shouldn't we use [`getForcProject`](https://github.com/FuelLabs/fuels-ts/blob/af2500bdbefbbf7236558426d55d061186867167/packages/utils/src/test-utils/getForcProject.ts#L47) to get all forc-related stuff in this file?